editor/src/components/canvas/canvas-strategies/strategies/absolute-reparent-strategy.tsx

```diff
@@ -254,6 +260,7 @@ export function createAbsoluteReparentAndOffsetCommands(
   projectContents: ProjectContentTreeRoot,
   nodeModules: NodeModules,
   forcePins: ForcePins,
+  containLayout: 'should-add' | 'should-not-add',
```

Comment:
Given how it's used, may I suggest renaming these values to something that indicates more specifically what is or is not to be added? (and moving it to a dedicated type so it can be reused in e.g. `getAbsoluteReparentPropertyChanges`).
Thinking of something like
```ts
type ShouldAddContainLayout = 'add-contain-layout' | 'dont-add-contain-layout'
```
but whatever makes sense to you would work.

Reply:
Yeah, that's a better naming, change made in 9fe611e74.